src/styles/ThemeStyles.ts

```diff
@@ -9,6 +9,7 @@ export const theme = createMuiTheme({
     },
     secondary: {
       main: '#757575',
+      light: '#64DD17', /* active green */
```

Comment:
Since `secondary` is grey, it doesn't make sense to make the `light` variant green. Look into other colors that may fit better, like `info` or `success` https://material-ui.com/customization/palette/